src/main/java/capstone/examlab/questions/controller/QuestionsController.java

```diff
@@ -22,43 +25,50 @@
 @Slf4j
 @RequiredArgsConstructor
 @RestController
-@RequestMapping("api/v1/exams/")
+@RequestMapping("api/v1/")
```

Comment:
```suggestion
@RequestMapping("/api/v1")
```
이렇게 '/' 로 시작하는게 좋을 것 같은데, 제가 작성했던 exams 도 실수로 되어있네요

이 부분 통일하는게 좋을 것 같은데
Users 처럼 반드시 '/'로 시작하도록 바꾸는건 어떨까요?

Reply:
내용 확인했습니다 변경할게요